packages/schematics/src/utility/file-not-found-exception.ts

```diff
@@ -0,0 +1,6 @@
+export class FileNotFoundException extends Error {
```

Comment:
I think that there is already an Exception for that out-of-the-box;

https://github.com/angular/angular-cli/blob/master/packages/angular_devkit/core/src/exception/exception.ts#L22

```ts
export class FileDoesNotExistException extends BaseException {
  constructor(path: string) { super(`Path "${path}" does not exist.`); }
}
```